taca/nanopore/ONT_run_classes.py

```diff
@@ -65,9 +65,10 @@ def __init__(self, run_abspath: str):
                 self.rsync_options[k] = None
 
         # Get transfer details, depending on run type and instrument
-        self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
-            "instruments"
-        ][self.instrument]
+        if hasattr(self, "run_type"):
+            self.transfer_details = CONFIG["nanopore_analysis"]["run_types"][self.run_type][
```

Comment:
Does this need a default value as well? If `run_type` is not available?

Reply:
The run_type attribute should always be available if it's used, so I'm content with it throwing an error if that is ever the case